influxdb_client/client/influxdb_client.py

```diff
@@ -173,7 +175,8 @@ def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gz
             }
 
         """
-        return InfluxDBClient._from_config_file(config_file=config_file, debug=debug, enable_gzip=enable_gzip, **kwargs)
+        return InfluxDBClient._from_config_file(config_file=config_file, config_name=config_name,
+                                                debug=debug, enable_gzip=enable_gzip, **kwargs)
```

Comment:
Please use `kwargs` instead of new argument:

```suggestion
        return InfluxDBClient._from_config_file(config_file=config_file, config_name=config_name, enable_gzip=enable_gzip, **kwargs)
```